### PR TITLE
default mount type for prod & reinstall mountdir if fail on darwin

### PR DESCRIPTION
**Pull request by songgao**
This supersedes https://github.com/keybase/kbfs/pull/1291 which was reverted.

### CODE REVIEW

**Comment by songgao** [comment]
@strib have you had a chance to look at this yet?

**Comment by strib** [comment]
Nope.

File: kbfsfuse/main.go [hunk 1]
```diff
@@ -30,13 +30,13 @@ const usageFormatStr = `Usage:
 
 To run against remote KBFS servers:
   kbfsfuse
-    [-runtime-dir=path/to/dir] [-label=label] [-mount-type=force]
+    [-runtime-dir=path/to/dir] [-label=label] [-mount-type=` + defaultMountType + `]
```
Comment (strib) [inline kbfsfuse/main.go]: Should this be something like "-mount-type=[default|force|none]"?

Comment (songgao) [reply kbfsfuse/main.go]: sure; will update!

**Pull request merged**